py/selenium/common/exceptions.py

```diff
@@ -38,7 +39,7 @@ def __str__(self) -> str:
         exception_msg = f"Message: {self.msg}\n"
         if self.screen:
             exception_msg += "Screenshot: available via screen\n"
-        if self.stacktrace:
+        if self.stacktrace and os.getenv("INCLUDE_STACKTRACE"):
```

Comment:
```suggestion
        if self.stacktrace and not os.getenv("EXCLUDE_STACKTRACE"):
```
Could you let me know if this change has been accepted?